pkg/convert/testdata/programs/builtin_functions/pcl/main.pp

```diff
@@ -1396,10 +1396,10 @@
 
 # Examples for tolist
 output "funcTolist0" {
-  value = notImplemented("tolist([\"a\",\"b\",\"c\"])")
+  value = ["a", "b", "c"]
 }
 output "funcTolist1" {
-  value = notImplemented("tolist([\"a\",\"b\",3])")
+  value = ["a", "b", 3]
```

Comment:
This is actually incorrect per:

https://developer.hashicorp.com/terraform/language/functions/tolist

```
Since Terraform's concept of a list requires all of the elements to be of the same type, mixed-typed elements will be converted to the most general type
```

However in the vast majority of cases for the examples use case this level of refinement is not needed as the passed in elements are uniformly typed.

I'd like to backlog this and proceed, if possible.